components/collection/drop/MintSection.vue

```diff
@@ -123,6 +124,10 @@ const emit = defineEmits(['mint'])
 
 const { urlPrefix } = usePrefix()
 
+const loading = computed(
```

Comment:
it isn't realy `loading`
more like `readyToDisplay` i guess

Reply:
it's used for the button state 
![CleanShot 2023-12-27 at 14 34 55@2x](https://github.com/kodadot/nft-gallery/assets/44554284/4c7a5dc7-9c2f-4fea-a488-5b1f39d91d34)

Reply:
not  a fan of the name, but it's minor so ill let it go